packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/ListView/tests/index.test.js

```diff
@@ -17,13 +26,23 @@ jest.mock('@strapi/helper-plugin', () => ({
 
 const history = createMemoryHistory();
 
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+    },
+  },
+});
+
 const App = (
   <ThemeProvider theme={lightTheme}>
-    <IntlProvider locale="en" messages={{}} defaultLocale="en" textComponent="span">
-      <Router history={history}>
-        <ListView />
-      </Router>
-    </IntlProvider>
+    <QueryClientProvider client={queryClient}>
+      <IntlProvider locale="en" messages={{}} defaultLocale="en" textComponent="span">
+        <Router history={history}>
```

Comment:
They actually recommend using `MemoryRouter` for testing which means you don't need to import or create `history`